deployment/v3/utils/prop_migrator/knowledge/latest-Value-takes-priority.csv

```diff
@@ -325,82 +284,6 @@ resident-default.properties,CREDENTIAL_TYPES_URL
-application-default.properties,mosip.kernel.crypto.symmetric-algorithm-name
-application-default.properties,mosip.kernel.crypto.asymmetric-algorithm-name
-application-default.properties,mosip.kernel.crypto.gcm-tag-length
-application-default.properties,mosip.kernel.crypto.hash-symmetric-key-length
-application-default.properties,mosip.kernel.crypto.hash-algorithm-name
-application-default.properties,mosip.kernel.crypto.sign-algorithm-name
-application-default.properties,mosip.kernel.crypto.hash-iteration
-application-default.properties,mosip.kernel.data-key-splitter
-application-default.properties,mosip.kernel.signature.signature-request-id
-application-default.properties,mosip.kernel.signature.signature-version-id
-application-default.properties,mosip.kernel.prid.restricted-numbers
-application-default.properties,mosip.kernel.prid.length
-application-default.properties,mosip.kernel.prid.sequence-limit
-application-default.properties,mosip.kernel.prid.repeating-block-limit
-application-default.properties,mosip.kernel.prid.repeating-limit
-application-default.properties,mosip.kernel.prid.not-start-with
-application-default.properties,mosip.kernel.uin.length
-application-default.properties,mosip.kernel.uin.restricted-numbers
-application-default.properties,mosip.kernel.uin.length.repeating-block-limit
-application-default.properties,mosip.kernel.uin.length.sequence-limit
-application-default.properties,mosip.kernel.uin.length.repeating-limit
-application-default.properties,mosip.kernel.uin.length.conjugative-even-digits-limit
-application-default.properties,mosip.kernel.uin.length.reverse-digits-limit
-application-default.properties,mosip.kernel.uin.length.digits-limit
-application-default.properties,mosip.kernel.vid.restricted-numbers
-application-default.properties,mosip.kernel.vid.not-start-with
-application-default.properties,mosip.kernel.vid.length.repeating-limit
-application-default.properties,mosip.kernel.vid.length.repeating-block-limit
-application-default.properties,mosip.kernel.vid.length.sequence-limit
-application-default.properties,mosip.kernel.vid.length
-application-default.properties,mosip.kernel.registrationcenterid.length
-application-default.properties,mosip.kernel.machineid.length
-application-default.properties,mosip.kernel.rid.length
-application-default.properties,mosip.kernel.rid.timestamp-length
-application-default.properties,mosip.kernel.rid.sequence-length
-application-default.properties,mosip.kernel.otp.expiry-time
-application-default.properties,mosip.primary-language
-application-default.properties,mosip.secondary-language
-application-default.properties,mosip.kernel.applicant.type.age.limit
-application-default.properties,mosip.idrepo.identity.allowedBioAttributes
-application-default.properties,mosip.idrepo.identity.bioAttributes
+registration-processor-default.properties,registration.processor.reprocess.minutes
+registration-processor-default.properties,registration.processor.queue.username
```

Comment:
As per the comment and delete marking these both to be deleted

Reply:
updated sir.